core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -169,6 +185,7 @@ private[log] class LogCleanerManager(val logDirs: Array[File], val logs: Pool[To
           state match {
             case LogCleaningPaused =>
               inProgress.remove(topicAndPartition)
+              decrementNumFilthyLogs()
```

Comment:
This is decrementing just the metric "num-filthy-logs" that was newly introduced in this PR. It's not used to drive any cleaning logic.